binder/postBuild

```diff
@@ -0,0 +1,2 @@
+pip install git+git://github.com/tomlouden/SPIDERMAN.git@69911b042bc46615ec9b39048a69e0d77c8542ad
+pip install -e.
```

Comment:
The `postBuild` file is executed as a script at the end of the docker image building process. This is how starry get's installed.

Reply:
👆 @rodluger